### PR TITLE
added zack's article

### PR DESCRIPTION
**Pull request by omfgnuts**
Zack's article preview

### CODE REVIEW

File: assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc [hunk 1]
```diff
@@ -0,0 +1,465 @@
+= webpack Plugin written by ChatGPT
+
+== Introduction
+
+No matter what social media platform you are on if you follow the tech space, or any space for that matter, you are most likely seeing content about ChatGPT or even content generated by ChatGPT.
+
+When it first came out a few weeks ago, I played with it, thought it was pretty cool and had my kids started using it to write guided stories with ideas from their imagination. When combined with MidJourney AI generated art the overall experience was quite amazing.
+
+Last night seeing https://dev.to/scriptedalchemy[@scriptedalchemy] share about his https://www.linkedin.com/feed/update/urn:li:activity:7009395526903234560/[experiences] with it I decided to try something a little more challenging.
```
Comment (Alexveselun) [inline assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc:9]: Links should be opened on the new tab

File: assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+npx webpack
+----
+
+That's it! You have now created a webpack plugin in TypeScript that wraps module federation and adds additional logging.
+
+____
+
+== Modifying the results
+
+Seeing what the results were it was quite verbose, but it does not follow the platform I usually like to use, https://dev.to/nrwl_io[@nrwl_io's] Nx. So I did a redirect on the conversation.
```
Comment (Alexveselun) [inline assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc:204]: Links should be opened on the new tab

**Review by Alexveselun — CHANGES_REQUESTED** [review]

File: assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc (hunk 1 above)
Comment (omfgnuts) [reply assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc:9]: Cmd + Click :D Or mouse wheel click?

Comment (Alexveselun) [reply assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc:9]: or add to the link "^" attribute :D  example: [experiences^]

**Review by Alexveselun — APPROVED** [review]

**Comment by Alexveselun** [comment]
Testing completed. No critical issues were found

**Pull request merged**